src/patient-registration/patient-registration.component.tsx

```diff
@@ -257,33 +240,6 @@ export const PatientRegistration: React.FC = () => {
     }
   }, [existingPatient]);
 
-  const getNames = (values: FormValues) => {
```

Comment:
a lot of my changes involve migrating a lot of this logic into it's module since there's no reason to define logic insdie the component itself. also creates a weird a dependency management situation as well as making the component harder to unit test